File: Beginner/1012 (Area)/1012.cs
using System;

namespace URI
{
    class Program
    {
        static void Main(string[] args)
        {
            string[] texto = Console.ReadLine().Split(' ');
            double a = double.Parse(texto[0]), b = double.Parse(texto[1]), c = double.Parse(texto[2]);
            double result = (a * c) / 2;
            Console.WriteLine("TRIANGULO: ", string.Format("{0:0.000}", result));
            result = 3.14159 * c * c;
            Console.WriteLine("CIRCULO: ", string.Format("{0:0.000}", result));
            result = ((a + b) * c) / 2;
            Console.WriteLine("TRAPEZIO: ", string.Format("{0:0.000}",result));
            result = b * b;
            Console.WriteLine("QUADRADO: ", string.Format("{0:0.000}", result));
            result = a * b;
            Console.WriteLine("RETANGULO: ", string.Format("{0:0.000}", result ));
        }
    }
}

File: Beginner/1012 (Area)/1012.java
import java.io.IOException;
import java.util.Scanner;
import java.text.DecimalFormat;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner ler = new Scanner(System.in);
        DecimalFormat df = new DecimalFormat("0.000");

        double a = ler.nextDouble();
        double b = ler.nextDouble();
        double c = ler.nextDouble();

        System.out.println("TRIANGULO: " + df.format((a * c)/2));
        System.out.println("CIRCULO: " + df.format(3.14159 * c * c));
        System.out.println("TRAPEZIO: " + df.format(((a+b)*c)/2));
        System.out.println("QUADRADO: " + df.format(b*b));
        System.out.println("RETANGULO: " + df.format(a*b));
        
    }
 
}